pages/common/gcalcli.md

```diff
@@ -0,0 +1,24 @@
+# gcalcli
+
+> Command line tool to interact with Google Calendar.
+> First time you run it launches request for API authorization.
```

Comment:
```suggestion
> Requests Google API authorization upon first launch.
> More information: <https://github.com/insanum/gcalcli>.
```
Sounds better IMHO, what do you think? Also, let's add an homepage while we're at it.